prime-router/src/test/kotlin/fhirengine/azure/FHIRConverterIntegrationTests.kt

```diff
@@ -1142,4 +1142,33 @@ class FHIRConverterIntegrationTests {
             assertThat(report.bodyFormat).isEqualTo("HL7")
         }
     }
+
+    @Test
+    fun `test should gracefully handle a case with an unsupported contents`() {
```

Comment:
This test already exists as part of: `should successfully convert HL7 messages`?

This is that tests setup code:
```
val receivedReportContents =
            listOf(cleanHL7Record, invalidHL7Record, unparseableHL7Record, badEncodingHL7Record)
                .joinToString("\n")
```
notice: invalidHL7Record, unparseableHL7Record, badEncodingHL7Record